apps/emqx_connector/src/emqx_connector_ssl.erl

```diff
@@ -60,6 +63,28 @@ clear_certs(RltvDir, #{ssl := OldSSL} = _Config) ->
 clear_certs(_RltvDir, _) ->
     ok.
 
+try_clear_certs(RltvDir, #{<<"connector">> := NewConnector}, #{<<"connector">> := OldConnector}) when
```

Comment:
There should be a help function to normalize all keys to binary.
Then we only need to implement one flavor of the functions